src/features/blacksmith/components/Crafting.tsx

```diff
@@ -28,6 +29,10 @@ export const Crafting: React.FC<Props> = ({ onClose }) => {
             <img src={nft} className="h-5 mr-2" />
             <span className="text-sm text-shadow">Items</span>
           </Tab>
+          <Tab isActive={tab === "foods"} onClick={() => setTab("foods")}>
+            <img src={food} className="h-5 mr-2" />
+            <span className="text-sm text-shadow">Foods</span>
```

Comment:
```suggestion
            <span className="text-sm text-shadow">Food</span>
```

In this case, I believe 'Food' can be used as the plural. It sounds a bit weird using `Foods`

Reply:
Okay I'll work on it later today